lib/3scale_toolbox/commands/backend_command/copy_command.rb

```diff
@@ -0,0 +1,64 @@
+require '3scale_toolbox/commands/backend_command/copy_command/task'
+require '3scale_toolbox/commands/backend_command/copy_command/create_or_update_target_backend_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_metrics_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_methods_task'
+require '3scale_toolbox/commands/backend_command/copy_command/copy_mapping_rules_task'
+
+module ThreeScaleToolbox
+  module Commands
+    module BackendCommand
+      class CopySubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'copy'
+            usage       'copy [opts] -s <source_remote> -d <target_remote> <source_backend>'
```

Comment:
Can we use dashes instead of underscore when there are keywords with multiple words? (source-remote instead of source_remote). I've seen we try to generally do that in this codebase. See for example copy service command or metrics create command.
In case we modify this don't forget to update the markdown documentation.

Reply:
:+1: